README.md

```diff
@@ -100,6 +100,29 @@ With this new connector, you should be able to simply install onto a cluster (ne
 
 See [Issue #26](https://github.com/microsoft/sql-spark-connector/issues/26) for more details.
 
+### Executing custom SQL through the connector
+The previous Azure SQL Connector for Spark provided the ability to execute custom SQL code like DML or DDL statements through the connector. This functionality is out-of-scope of this connector since it is based on the DataSource APIs. This functionality is readily provided by libraries like pyodbc or you can use the standard java sql interfaces as well.
+
+@B4PJS has provided examples using the standard java sql interfaces in [Issue #21](https://github.com/microsoft/sql-spark-connector/issues/21)
+
+```python
```

Comment:
Should not provide an usage example here. It not the connector context.

Good to just state that direct DML/DDL is not supported or a feature for connector. We can close the issues with the connect and an example, but not the sample in the connector readme.